website/docs/routes/new-media.md

```diff
@@ -327,6 +327,17 @@ Category of news
 
 </Route>
 
+
+### Military - Finance News 财经 - 财经新闻 {#china.com-zhong-hua-wang-military-finance-news-cai-jing-cai-jing-xin-wen}
+
+<Route author="KingJem" example="/china/finance/" path="/china/finance/:category?/:limit?" paramsDesc={['Category of news. See the form below for details, default is suggest news.','limit default is 30']}>
```

Comment:
https://github.com/DIYgod/RSSHub/blob/156903a9dc70aee8805008650ffc5c4a29daee99/lib/v2/china/finance/finance.js#L26
`limit` is not a path parameter.